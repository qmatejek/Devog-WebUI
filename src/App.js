import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { Home } from './Home';
import { NoMatch } from './NoMatch';
import {NavBar} from './components/NavBar'
import { Login } from './Login';
import { Video } from './components/Video';
import './CSS/styles.css';

function App() {
  return (
    <React.Fragment>
      <NavBar/>
        <Router>
          <Switch>
            <Route exact path="/" component = {Home}/>
            <Route path="/login" component = {Login}/>
            <Route path="/video" component = {Video}/>
            <Route component = {NoMatch}/>
          </Switch>
        </Router>
    </React.Fragment>
  );
}

export default App;
