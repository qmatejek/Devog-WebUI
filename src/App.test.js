import React from 'react';
//import { render } from '@testing-library/react';
import App from './App';
import { shallow, render } from 'enzyme';

// test('renders learn react link', () => {
//   const { getByText } = render(<App />);
//   const linkElement = getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });

it("renders correctly", () => {
  const wrapper = shallow(
    <App />
  );

  expect(wrapper).toMatchSnapshot();
});