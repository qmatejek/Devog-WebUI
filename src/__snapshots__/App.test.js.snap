// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <NavBar />
  <BrowserRouter>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/login"
      />
      <Route
        component={[Function]}
        path="/video"
      />
      <Route
        component={[Function]}
      />
    </Switch>
  </BrowserRouter>
</Fragment>
`;
